metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "puppetlabs-cron_core",
   "version": "1.0.0",
-  "author": "Puppet Labs",
+  "author": "puppetlabs",
```

Comment:
Should the author just be "Puppet"? I'm not sure what this is supposed to map to, TBH...

Reply:
Nope, this is correct. It should map to the Forge namespace which is still `puppetlabs`